Apps/MAPL_GridCompSpecs_ACG.py

```diff
@@ -249,6 +256,7 @@ def open_with_header(filename):
     f.write(header())
     return f
 
+categories = ("IMPORT","EXPORT","INTERNAL")
```

Comment:
Is there a pythonic way to make this appear more as a global const?   E.g., should it maybe be in all caps?   Not a recommendation, just a question.

Reply:
Change made